package.json

```diff
@@ -16,8 +16,9 @@
     "scripts": {
         "dev": "cross-env npm run build && node ./scripts/qode.js dist/demo.js",
         "build": "cross-env tsc && npm run build:addon",
-        "install": "cross-env npm run setupqt && (node ./scripts/skip.js || npm run build:addon)",
+        "install": "cross-env npm run setupqt && (node ./scripts/skip.js || npm run setupbinary)",
```

Comment:
you can add an or condition here to build the binary if prevuild installation fails

Reply:
sure, I can do that.